packages/strapi-generate-new/lib/utils/db-client-dependencies.js

```diff
@@ -16,7 +16,7 @@ module.exports = ({ scope, client }) => {
     case 'mysql':
       return {
         'strapi-connector-bookshelf': scope.strapiVersion,
-        knex: '<0.20.0',
+        knex: '^0.20.0',
```

Comment:
we are using bookshelf v 1.0.1 and the knex dependecy is  "knex": ">=0.15.0 <0.20.0" I don't think we should change this without upgrade bookshelf first

Reply:
Better to look at updating bookshelf?

Reply:
Yep that was what I meant :) we should upgrade bookshelf and knex at once

Reply:
@XuluWarrior I don't see any breaking changes in bookshelf v1.2.0 can you please upgrade that as well and we can let TravisCI do some tests: https://github.com/bookshelf/bookshelf/blob/master/CHANGELOG.md

Reply:
The peer dependency in bookshelf v1.2.0 is `">=0.15.0 <0.21.0"`
Should the peer dependency in https://github.com/strapi/strapi/blob/77be3ea4c596543d63e244a3bd7e92b486e2ba23/packages/strapi-connector-bookshelf/package.json#L29 match?
Should the dependency here (db-client-dependencies.js) actually use `~` rather than `^`?
Otherwise, a future 0.22.x release could be installed before it is supported by bookshelf.

Reply:
you can set bookshelf to 1.2.0 and knex to ~0.21.0 like bookshelf does in their devDependencies